packages/core/src/index.ts

```diff
@@ -2,14 +2,20 @@ export { buildModule } from "./buildModule";
 export * from "./errors";
 export * from "./initialization";
 export { defineModule } from "./new-api/define-module";
+export { Deployer } from "./new-api/deployer";
 /* TODO: how is module constructor getting exposed? */
 export { ModuleConstructor } from "./new-api/internal/module-builder";
+export { FileJournal, MemoryJournal } from "./new-api/journal";
```

Comment:
Should be private

Reply:
Tracked by: https://github.com/NomicFoundation/ignition/issues/273